File: Coroutines/app/src/main/java/com/kantek/coroutines/models/LoginResponse.java
package com.kantek.coroutines.models;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "loginResponse", strict = false)
public class LoginResponse {
    @Element(name = "loginReturn", type = String.class)
    private String loginReturn;

    public String getLoginReturn() {
        return loginReturn;
    }

    public void setLoginReturn(String loginReturn) {
        this.loginReturn = loginReturn;
    }
}

File: Coroutines/app/src/main/java/com/kantek/coroutines/models/LoginResponse.kt
package com.kantek.coroutines.models

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "loginResponse", strict = false)
class LoginResponse {
    @get:Element(name = "loginReturn", type = String::class)
    @set:Element(name = "loginReturn", type = String::class)
    var loginReturn: String? = null
}
